src/rest_framework_dso/fields.py

```diff
@@ -231,6 +235,18 @@ def apply(
                 code="fields",
             )
 
+        # We keep the order of the _fields parameter that has been passed in.
+        if self._ordered_fields:
```

Comment:
Ik dacht heel even dat deze altijd true zou zijn, maar dat is natuurlijk niet zo bij een exclude... Misschien even comment?